actor/src/main/resources/reference.conf

```diff
@@ -739,6 +739,9 @@ pekko {
     # to reduce noise.
     warn-on-no-serialization-verification = on
 
+    # list of fqcn of classes that may use non-pekko serializer within pekko package without warn log
+    warn-non-pekko-serializer-whitelist = ${?akka.actor.warn-non-pekko-serializer-whitelist} []
```

Comment:
Could we call this an `allow-list` instead of `whitelist` - to avoid the connotations about colors?

Reply:
sure...